tests/foreman/virtwho/cli/test_esx_sca.py

```diff
@@ -0,0 +1,543 @@
+
+        :expectedresults: debug option can be updated.
+
+        :CaseLevel: Integration
+
+        :CaseImportance: Medium
+        """
+        assert virtwho_config['name'] == form_data['name']
+        options = {'true': '1', 'false': '0', 'yes': '1', 'no': '0'}
+        for key, value in sorted(options.items(), key=lambda item: item[0]):
+            target_sat.cli.VirtWhoConfig.update({'id': virtwho_config['id'], 'debug': key})
+            command = get_configure_command(virtwho_config['id'], module_sca_manifest_org.name)
+            deploy_configure_by_command(
+                command, form_data['hypervisor-type'], org=module_sca_manifest_org.label
+            )
+            assert get_configure_option('debug', ETC_VIRTWHO_CONFIG) == value
+
+    @pytest.mark.tier2
+    def test_positive_name_option(
+        self, module_sca_manifest_org, form_data, virtwho_config, target_sat
+    ):
+        """Verify debug option by hammer virt-who-config update"
+
+        :id: 8d22c7b8-756b-4f79-83be-34ccb2609388
+
+        :expectedresults: name option can be updated.
+
+        :CaseLevel: Integration
+
+        :CaseImportance: Medium
+        """
+        assert virtwho_config['name'] == form_data['name']
+        new_name = gen_string('alphanumeric')
+        target_sat.cli.VirtWhoConfig.update({'id': virtwho_config['id'], 'new-name': new_name})
+        virt_who_instance_name = target_sat.cli.VirtWhoConfig.info({'id': virtwho_config['id']})[
+            'general-information'
+        ]['name']
+        assert virt_who_instance_name == new_name
+        target_sat.cli.VirtWhoConfig.update(
+            {'id': virtwho_config['id'], 'new-name': form_data['name']}
+        )
+
+    @pytest.mark.tier2
+    def test_positive_interval_option(
+        self, module_sca_manifest_org, form_data, virtwho_config, target_sat
+    ):
+        """Verify interval option by hammer virt-who-config update"
+
+        :id: 9b2ffa5a-c0e9-43e7-bf63-f64832cf7715
+
+        :expectedresults: interval option can be updated.
+
+        :CaseLevel: Integration
+
+        :CaseImportance: Medium
+        """
+        options = {
+            '60': '3600',
+            '120': '7200',
+            '240': '14400',
+            '480': '28800',
+            '720': '43200',
+            '1440': '86400',
+            '2880': '172800',
+            '4320': '259200',
+        }
+        for key, value in sorted(options.items(), key=lambda item: int(item[0])):
+            target_sat.cli.VirtWhoConfig.update({'id': virtwho_config['id'], 'interval': key})
+            command = get_configure_command(virtwho_config['id'], module_sca_manifest_org.name)
+            deploy_configure_by_command(
+                command, form_data['hypervisor-type'], org=module_sca_manifest_org.label
+            )
+            assert get_configure_option('interval', ETC_VIRTWHO_CONFIG) == value
+
+    @pytest.mark.tier2
+    def test_positive_filter_option(
```

Comment:
Same question about breaking this into parametrized tests.

Reply:
> Same question about breaking this into parametrized tests.

@JacobCallahan Have already parametrized tests.
```
(robottelo_vv) [root@dell-per740-68-vm-04 robottelo_20230307]# pytest  ./tests/foreman/virtwho/cli/test_esx_sca.py  -k test_positive_filter_option  --disable-pytest-warnings -q
....                                                                                                                                                                                                        [100%]
4 passed, 12 deselected, 7 warnings in 320.71s (0:05:20)

```